test/test_check_usernames.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import vcr
+
+from src.check_usernames import check
+
+@vcr.use_cassette('test/vcr_cassettes/check-success.yaml')
```

Comment:
So I'm not too familiar with vcrs but shouldn't the yaml file here have a more unique name? Perhaps with resembling the code hierarchy?

Reply:
I'm not really familiar with them either, I used `check-success` because the name of the function is `check()`, not the phrase "check success". I think the code hierarchy thing is a good idea so I'll make it `test/vcr_cassettes/check_usernames/check-success.yaml`

Reply:
The docs for vcr put them in a `fixtures` folder, which makes sense since they are indeed a type of generated test fixtures. Would `test/fixtures/vcr_cassettes/... ` make sense?

Reply:
just saw this, let me commit again lol